adapter/outbound/sudoku.go

```diff
@@ -163,12 +168,13 @@ func NewSudoku(option SudokuOption) (*Sudoku, error) {
 		return nil, fmt.Errorf("table-type must be prefer_ascii or prefer_entropy")
 	}
 
-	seed := option.Seed
-	if seed == "" {
-		seed = option.Key
+	seed := option.Key
+	if recoveredFromKey, err := crypto.RecoverPublicKey(option.Key); err == nil {
+		seed = crypto.EncodePoint(recoveredFromKey)
 	}
 
-	table := sudoku.NewTable(seed, tableType)
+	// Use local initTable instead of sudoku.NewTable to control logging
```

Comment:
Why not simply remove logging from sudoku.NewTable instead of copying it from outside?